assets/chezmoi.io/docs/developer/website.md

```diff
@@ -5,20 +5,33 @@ MkDocs](https://squidfunk.github.io/mkdocs-material/) from the contents of the
 `assets/chezmoi.io/docs/` directory. It hosted by [GitHub pages](https://pages.github.com/) from
 the [`gh-pages` branch](https://github.com/twpayne/chezmoi/tree/gh-pages).
 
-Install Material for MkDocs and the required plugins with:
+Change into the website directory:
 
 ```console
-$ pip3 install mkdocs-material mkdocs-mermaid2-plugin mkdocs-redirects mkdocs-simple-hooks
+$ cd assets/chezmoi.io
+```
+
+(Optional) Create a virtualenv with:
```

Comment:
I would say `(Recommended)` next to this. Otherwise, the `pip3` command should be `pip3 install --user -r requirements.txt`.

Reply:
I've just added both methods via tabs. Let me know if you think it looks too busy...